pkg/operator/hive/hive.go

```diff
@@ -173,6 +174,30 @@ func (r *ReconcileHiveConfig) deployHive(hLog log.FieldLogger, h *resource.Helpe
 
 	}
 
+	// Remove old validatingwebhookconfigurations
+	removeValidatingWebhooks := []string{
+		"clusterdeployments.admission.hive.openshift.io",
+		"clusterimagesets.admission.hive.openshift.io",
+		"dnszones.admission.hive.openshift.io",
+		"selectorsyncsets.admission.hive.openshift.io",
+		"syncsets.admission.hive.openshift.io",
+	}
+	for _, webhookName := range removeValidatingWebhooks {
+		webhookConfig := &admissionregistrationv1beta1.ValidatingWebhookConfiguration{}
+		err := r.Get(context.Background(), types.NamespacedName{Name: webhookName}, webhookConfig)
+		if err != nil {
+			hLog.Infof("Validating webhook does not exist: %v")
```

Comment:
Need to check the type of error here as the log message could be a bit inaccurate. If it doesn't exist don't log anything. Any other error should probably be logged at log.error.